platform/mellanox/docker-syncd-mlnx/lib/port_utils.py

```diff
@@ -48,7 +82,9 @@ def sx_get_ports_map(handle):
             lane_index = get_lane_index(lane_bmap, ASIC_MAX_LANES[chip_type])
             assert lane_index != -1, "Failed to calculate port index"
             
-            sonic_index = label_port * ASIC_MAX_LANES[chip_type] + lane_index;
+            first_lane = label_port * ASIC_MAX_LANES[chip_type] + lane_index;
```

Comment:
Need to make sure that we have mapping for SPC4 chip, and it will be better to have this table in dynamic way so in the future, for new chip, we will not need to update the table

Reply:
SPC4 is in the mapping.

Reply:
I verified, map contains SPC4.